### ISSUE TITLE
Have you tried to add weight decay factor?

### ISSUE THREAD

**Issue by BKZero** [mariolew/Deep-Alignment-Network-tensorflow#11]
RT
i do not found weight decay in all three implementation. Did i miss something?
